clap-utils/src/keypair.rs

```diff
@@ -215,6 +195,23 @@ impl std::fmt::Debug for SignerSourceKind {
     }
 }
 
+impl SignerSourceKind {
+    pub fn new_filepath_checked(path: String) -> Result<Self, SignerSourceError> {
+        std::fs::metadata(&path)
+            .map_err(|_| {
+                std::io::Error::new(
+                    std::io::ErrorKind::Other,
+                    format!(
+                    "No default signer found, run \"solana-keygen new -o {}\" to create a new one",
```

Comment:
This message no longer only applies to the "default", but any singer source.

Reply:
Good point! Not sure if I should generalize the message or revisit my first pass that deferred the `stat()` call until the first query :thinking:

Reply:
I went with the latter